ui/src/json/feeds.json

```diff
@@ -0,0 +1,149 @@
+[
```

Comment:
This file looks suspicious since not all hydrophones are in this json (e.g., Point Robinson).
Can it be deleted?  Or does it just need to be updated?